### ISSUE TITLE
SSD Input Resize

### ISSUE THREAD

**Issue by meenakshik1993** [tensorflow/models#4679]
Hello,
I am fine-tuning the SSD+MobilenetV2 pretrained model on my own dataset. Since the images in my dataset are a lot bigger and with a different aspect ratio than 300x300, I do not wish to resize them to this fixed size. Instead, I change the fixed_shape_resizer to resize my inputs to 400x720 and the model trains fine and the validation accuracy is even better than what I got after resizing it to 300x300. Now, I believe that SSD works with square images (because it needs square feature maps!). My question is how is the API resizing the inputs to make them square with my input configuration? Thanks!

Have I written custom code: NO
OS Platform and Distribution: N/A
TensorFlow installed from: N/A
TensorFlow version: N/A
Bazel version: N/A
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: N/A

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new/choose). Thanks!
